resources/airflow.rb

```diff
@@ -969,6 +969,13 @@ def self.create_nat_vent_objects(model, runner, obj_name_natvent, building, nat_
     avail_sch = OpenStudio::Model::ScheduleRuleset.new(model)
     avail_sch.setName(obj_name_natvent + " avail schedule")
 
+    schedule_type_limits = OpenStudio::Model::ScheduleTypeLimits.new(model)
+    schedule_type_limits.setName(avail_sch.name.to_s)
+    schedule_type_limits.setLowerLimitValue(0)
+    schedule_type_limits.setUpperLimitValue(1)
+    schedule_type_limits.setNumericType("Discrete")
+    avail_sch.setScheduleTypeLimits(schedule_type_limits)
```

Comment:
Can you use the Schedule.createScheduleTypeLimits method here?

Reply:
Yes.